### PR TITLE
Automatically add coord bounds for image plotting.

### PR DESCRIPTION
**Pull request by rhattersley**
Closes CubeBrowser/cube-explorer#63.

### CODE REVIEW

File: geoviews/plotting/__init__.py [hunk 1]
```diff
@@ -145,7 +146,18 @@ class GeoImagePlot(GeoPlot, ColorbarPlot):
...
+        # Make sure both coordinates have bounds to avoid iris warning.
+        for coord in cube.dim_coords:
+            if not coord.has_bounds():
+                coord.guess_bounds()
+                # Clip auto-generated latitude bounds to +/-90 to avoid
+                # cartopy warning.
+                if 'latitude' in coord.name():
+                    points = coord.points
+                    if (points >= -90).all() and (points <= 90).all():
+                        coord.bounds = np.clip(coord.bounds, -90, 90)
```
Comment (rhattersley) [inline geoviews/plotting/__init__.py]: @bjlittle, @pelson - how about we pull this upstream into Iris?

Comment (pelson) [inline geoviews/plotting/__init__.py]: Thorny one... but pragmatically, sounds sensible...

Comment (rhattersley) [inline geoviews/plotting/__init__.py]: See SciTools/iris#1978

Comment (rhattersley) [inline geoviews/plotting/__init__.py]: @jlstevens & @philippjfr - if the change makes it in upstream then do you want to wait for a new Iris release and simply leave this code out of geoviews? Or do you want this to work with older (e.g. current) versions of Iris?

Comment (rhattersley) [inline geoviews/plotting/__init__.py]: As pointed out by @philippjfr, geoviews will need a new release of Iris anyway, so let's hold off on merging this until we know whether the Iris PR can make it into that release.

Comment (rhattersley) [inline geoviews/plotting/__init__.py]: The Iris PR has been merged and it'll turn up in the next release, so I need to delete the code from here.

**Comment by philippjfr** [comment]
Looks good to me. 👍

**Pull request merged**